data/fixtures/recorded/languages/markdown/changeCell.yml

```diff
@@ -0,0 +1,27 @@
+languageId: markdown
+command:
+  version: 7
+  spokenForm: change cell
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: notebookCell}
```

Comment:
NB: `notebookCell` just means the "cell" scope type. It doesn't actually mean a notebook cell (and in fact here it means a markdown table cell), but we added notebooks first and we don't want to rename it.